### DISCUSSION TITLE
Whats in your mind?

### DISCUSSION THREAD

**Discussion by omegaui** [omegaui/omegaide#14] [Show and tell]
See Here

**Comment by omegaui** [comment]
Hi!

**Reply by mr-cheffy** [reply]
Hi!

**Comment by mr-cheffy** [comment]
So... I have some questions...

1. Is there a way to resize the window?
2. What does `Faster terminal mean`?
3. At the top-left of the window, when I hover an icon a lot of icons start to appear... why?

**Reply by omegaui** [reply]
Just an animation, to show how many languauge build loggers are supported

**Reply by mr-cheffy** [reply]
And what if you end up with 1000 languages?

**Reply by omegaui** [reply]
No, I will show only some of them!

**Reply by omegaui** [reply]
It just kinda looks cool to me

**Comment by omegaui** [comment]
No!
I haven't implemented the 1.part!
I am using Jeditem to emulate the System Terminal which is heavy
So the Faster Terminal is just a shell process watched by a JTextArea (thus, lighter)

**Reply by mr-cheffy** [reply]
I understand.

**Reply by omegaui** [reply]
But Using the SuperKey + Arrow (Left or Right) the window can be resized!

**Comment by mr-cheffy** [comment]
It takes a while to clone the repo...

**Reply by omegaui** [reply]
Yes, Its big around 150 MB in total

**Reply by mr-cheffy** [reply]
wow

**Comment by mr-cheffy** [comment]
How do I run it? (from the source)

**Reply by omegaui** [reply]
Just open the project with the current daily-build jar and hit run!

**Reply by mr-cheffy** [reply]
from the command line?

**Reply by omegaui** [reply]
No

**Reply by omegaui** [reply]
Before Launching the IDE
Replace the contents of .projectInfo with this
```
>JDK Path
-/usr/lib/jvm/jdk-17
>Main Class
-omega.IDE
>Non-Java Project
-false
>Language Tag
-0
>Project Classpath : Required Jars
-project-root$/lib/flatlaf-0.46.jar
-project-root$/lib/annotations.jar
-project-root$/lib/guava-14.0.1.jar
-project-root$/lib/jediterm-pty-2.2.jar
-project-root$/lib/jna-platform.jar
-project-root$/lib/jna.jar
-project-root$/lib/log4j-1.2.14.jar
-project-root$/lib/purejavacomm-0.0.17.jar
-project-root$/lib/pty4j-0.7.2.jar
>Project Classpath : Required Resource Roots
-project-root$/res
```
Launch the IDE
and open the folder omegaide

**Reply by mr-cheffy** [reply]
oo... I understand

**Reply by mr-cheffy** [reply]
Would that affect the release?

**Reply by omegaui** [reply]
Mom is calling, I have to go!
Don't you have any social media app?

**Reply by mr-cheffy** [reply]
Yea, discord and WhatsApp. (but I don't want to give out my number)

**Reply by omegaui** [reply]
> Would that affect the release?

No it just sets up the classpath

**Reply by omegaui** [reply]
> Yea, discord

Ok link it here and I will be back in a while

**Reply by omegaui** [reply]
> Yea, discord and WhatsApp. (but I don't want to give out my number)

Ok, I understand,
Don't worry we will do things here only!

**Comment by omegaui** [comment]
But its somehow difficult to chat in here

**Reply by mr-cheffy** [reply]
do u want my discord?

**Reply by omegaui** [reply]
Yes,

**Reply by mr-cheffy** [reply]
`Maurin Triciclin#5628`

**Reply by mr-cheffy** [reply]
don't question the name...

**Reply by omegaui** [reply]
Let me log into Discord

**Reply by mr-cheffy** [reply]
ok...

**Reply by omegaui** [reply]
It says I need a server invite to start

**Reply by omegaui** [reply]
Wait...I entered the unregistered email .....

**Reply by omegaui** [reply]
Sent friend request
